utils/DeployedContractsHelpers.ts

```diff
@@ -328,45 +359,52 @@ export async function mintWeth(hre: HardhatRuntimeEnvironment, user: SignerWithA
   await weth.connect(user).deposit({ value: amount });
 }
 
-export async function getOnChainBtcEurPrice(hre: HardhatRuntimeEnvironment) {
-  const networkName = hre.network.name;
-
-  return getOnChainCryptoFiatPrice(hre, constants.EUR_USD_FEED_ADDRESS[networkName], constants.BTC_USD_FEED_ADDRESS[networkName]);
+export async function getOnChainBtcStablePrice(hre: HardhatRuntimeEnvironment, stableSymbol: string) {
+  return getOnChainCryptoFiatPrice(hre, stableSymbol, "BTC");
 }
 
-export async function getOnChainEthEurPrice(hre: HardhatRuntimeEnvironment) {
-  const networkName = hre.network.name;
-
-  return getOnChainCryptoFiatPrice(hre, constants.EUR_USD_FEED_ADDRESS[networkName], constants.ETH_USD_FEED_ADDRESS[networkName]);
+export async function getOnChainEthStablePrice(hre: HardhatRuntimeEnvironment, stableSymbol: string) {
+  return getOnChainCryptoFiatPrice(hre, stableSymbol, "ETH");
 }
 
 export async function getOnChainWethUsdPrice(hre: HardhatRuntimeEnvironment) {
-  const networkName = hre.network.name;
-
-  return getOnChainCryptoUSDPrice(hre, constants.ETH_USD_FEED_ADDRESS[networkName]);
+  return getOnChainCryptoUSDPrice(hre, "ETH");
 }
 
 export async function getOnChainWbtcUsdPrice(hre: HardhatRuntimeEnvironment) {
-  const networkName = hre.network.name;
-
-  return getOnChainCryptoUSDPrice(hre, constants.BTC_USD_FEED_ADDRESS[networkName]);
+  return getOnChainCryptoUSDPrice(hre, "BTC");
 }
 
-export async function getOnChainCryptoUSDPrice(hre: HardhatRuntimeEnvironment, cryptoUsdFeedAddress: string) {
-  const cryptoUsdFeed = (await hre.ethers.getContractAt("AggregatorV3Interface", formatAddress(hre, cryptoUsdFeedAddress))) as AggregatorV3Interface;
+export async function getOnChainCryptoUSDPrice(hre: HardhatRuntimeEnvironment, cryptoSymbol: "ETH" | "BTC") {
+  const networkName = hre.network.name;
+  const cryptoToUsdFeedAddress = _.get(constants, [`${cryptoSymbol}_USD_FEED_ADDRESS`, networkName]);
+  if (!cryptoToUsdFeedAddress) {
+    throw new Error(`There is no field to usd feed address "${cryptoSymbol}_USD_FEED_ADDRESS" on network ${networkName}`);
```

Comment:
The error was a bit confusing at first, how about:
```suggestion
    throw new Error(`On network ${networkName}, there is no price feed address set for ${cryptoSymbol}_USD_FEED_ADDRESS`);
```

same goes for the other place we have this error